lib/resty/core/phase.lua

```diff
@@ -0,0 +1,57 @@
+local ffi = require 'ffi'
+local C = ffi.C
+local base = require "resty.core.base"
+local tostring = tostring
+
+ffi.cdef[[
+int ngx_http_lua_ffi_get_phase(ngx_http_request_t *r, char **err)
+]]
+
+
+local _M = {
+    _VERSION = base.version
+}
+
+
+local errmsg = base.get_errmsg_ptr()
+local context_lookup = {
```

Comment:
@p0pr0ck5 You know that Lua table keys can be arbitrary numbers instead of strings?